tests/base.py

```diff
@@ -82,15 +82,16 @@ def inspect_method_signature(method: callable, reference: callable):
         method_signature = signature(method).parameters
         reference_signature = signature(reference).parameters
 
-        # All concrete implementations are expected to accept *args
+        # Concrete implementations should not accept *args because
+        # interface methods are defined to only have named parameters
         assert (
-            "args" in method_signature
-        ), f"{method.__module__}.{method.__name__} is not accepting *args parameter"
+            "args" not in method_signature
+        ), f"{method.__module__}.{method.__name__} should not accept *args parameter"
 
         # Verify that args is exactly *args
-        assert (
-            method_signature["args"].kind == Parameter.VAR_POSITIONAL
-        ), "wrong args argument, expected *args"
+        # assert (
```

Comment:
Should we remove instead of commenting out?

Reply:
Yes we should... and we did!